src/tools/gophish_complete.py

```diff
@@ -3,24 +3,24 @@
 """Complete a campaign in GoPhish and/or output a GoPhish campaign summary.
 
 Usage:
-  gophish-complete (--auto=CAMPAIGN_ID | --complete | --summary ) [--log-level=LEVEL] SERVER API_KEY
+  gophish-complete (--complete | --summary) [--Campaign=NAME] [--log-level=LEVEL] SERVER API_KEY
   gophish-complete (-h | --help)
   gophish-complete --version
 
 Options:
   API_KEY                   GoPhish API key.
   SERVER                    Full URL to GoPhish server.
-  -a --auto=CAMPAIGN_ID     Complete a specific campaign identified by CAMPAIGN_ID.
-  -c --complete             Display a list of campaigns to choose which to complete.
-  -s --summary              Choose a campaign from a list to output a summary.
+  -C --Campaign=NAME        GoPhish campaign name.
+  -c --complete             Complete a campaign within GoPhish.
+  -s --summary              Output a summary of a GoPhish campaign.
   -h --help                 Show this screen.
   --version                 Show version.
   -l --log-level=LEVEL      If specified, then the log level will be set to
                             the specified value.  Valid values are "debug", "info",
                             "warning", "error", and "critical". [default: info]
 
 NOTE:
-  * If the auto flag is not provided, all assessment campaigns will be listed to select from.
+  * If the campaign option with a campaign name is not provided, all assessment campaigns will be listed to select from.
```

Comment:
I think you can simply say `If a campaign name is not provided, all...`

Reply:
Fixed in commit 1ab0c81.